tests/integrations/conftest.py

```diff
@@ -77,14 +78,15 @@ async def _factory(contract_name, *args, **kwargs):
             )
         caller_address = kwargs["caller_address"]
         del kwargs["caller_address"]
-        deploy_bytecode = hex_string_to_bytes_array(
+        deploy_bytecode = hex_string_to_felt_packed_array(
             contract.constructor(*args, **kwargs).data_in_transaction
         )
 
         with traceit.context(contract_name):
-            tx = await kakarot.deploy(bytecode=deploy_bytecode).execute(
-                caller_address=caller_address
-            )
+            tx = await kakarot.deploy(
+                bytecode=deploy_bytecode,
+                original_bytecode_len=80,
```

Comment:
what is this 80?

Reply:
It was a test, I think I pushed by mistake